AdGuard Home

[ERROR] in line 97: exit code 4: while executing command wget -qL https://raw.githubusercontent.com/AdguardTeam/AdGuardHome/master/scripts/install.sh

![0403_04_57](https://user-images.githubusercontent.com/6137813/222886676-e23d78e0-39e6-4fe0-99b8-7031abd8bc92.png)


**Bug reports will be closed after 12 hours of NO activity.**

--- a/ct/adguard.sh
+++ b/ct/adguard.sh
@@ -54,12 +54,12 @@ function default_settings() {
 function update_script() {
 header_info
 if [[ ! -d /opt/AdGuardHome ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
+wget -qL https://static.adguard.com/adguardhome/release/AdGuardHome_linux_amd64.tar.gz
 msg_info "Stopping AdguardHome"
 systemctl stop AdGuardHome
 msg_ok "Stopped AdguardHome"
 
 msg_info "Updating AdguardHome"
-wget -qL https://static.adguard.com/adguardhome/release/AdGuardHome_linux_amd64.tar.gz
 tar -xvf AdGuardHome_linux_amd64.tar.gz &>/dev/null
 mkdir -p adguard-backup
 cp -r /opt/AdGuardHome/AdGuardHome.yaml /opt/AdGuardHome/data adguard-backup/
@@ -74,7 +74,7 @@ msg_ok "Started AdguardHome"
 msg_info "Cleaning Up"
 rm -rf AdGuardHome_linux_amd64.tar.gz AdGuardHome adguard-backup
 msg_ok "Cleaned"
-msg_ok "Update Successfull"
+msg_ok "Update Successful"
 exit
 }
 

